_maps/RandomRuins/BeachRuins/beach_crashed_starwalker.dmm

```diff
@@ -0,0 +1,5515 @@
+	dir = 8
+	},
+/turf/open/floor/wood{
+	icon_state = "wood-broken4"
+	},
+/area/ruin/beach/starwalker)
+"gJ" = (
+/obj/effect/turf_decal/techfloor{
+	dir = 6
+	},
+/obj/structure/table/reinforced,
+/obj/item/door_seal{
+	pixel_x = 6
+	},
+/obj/machinery/status_display/shuttle{
+	pixel_y = -32
+	},
+/obj/machinery/light_switch{
+	pixel_x = 20;
+	dir = 8;
+	pixel_y = -7
+	},
+/obj/machinery/light/broken,
+/obj/item/reagent_containers/pill/lsd{
+	pixel_y = -2;
+	pixel_x = -1
+	},
+/turf/open/floor/plasteel/tech/grid,
+/area/ruin/beach/starwalker)
+"gU" = (
+/obj/effect/decal/cleanable/dirt,
+/obj/item/kirbyplants{
+	icon_state = "plant-10"
+	},
+/turf/open/floor/pod/light,
+/area/ruin/beach/starwalker)
+"hd" = (
+/obj/effect/decal/cleanable/dirt,
+/obj/effect/decal/cleanable/blood/tracks{
+	dir = 5
+	},
+/turf/open/floor/plating/dirt,
+/area/ruin/unpowered)
+"hf" = (
+/obj/effect/turf_decal/industrial/outline/yellow,
+/obj/structure/grille,
+/turf/open/floor/engine/hull,
+/area/ruin/beach/starwalker)
+"hg" = (
+/obj/item/soap/deluxe,
+/obj/item/bikehorn/rubberducky,
+/obj/machinery/shower{
+	dir = 1
+	},
+/obj/structure/curtain,
+/obj/structure/sign/poster/contraband/random{
+	pixel_x = -32
+	},
+/turf/open/floor/plating/catwalk_floor,
+/area/ruin/beach/starwalker)
+"hi" = (
+/obj/effect/turf_decal/weather/dirt{
+	dir = 1
+	},
+/turf/open/floor/plating/beach/sand{
+	light_range = 2
+	},
+/area/ruin/unpowered)
+"hx" = (
+/mob/living/simple_animal/beachcarp/salmon{
+	dir = 4
+	},
+/obj/effect/decal/fakelattice{
+	icon_state = "lattice-3"
+	},
+/turf/open/water/beach,
+/area/ruin/unpowered)
+"hL" = (
+/obj/machinery/light{
+	dir = 4
+	},
+/obj/structure/closet/crate/trashcart,
+/obj/item/trash/can/food/beans{
+	pixel_x = 4;
+	pixel_y = -5
+	},
+/obj/item/trash/popcorn,
+/obj/item/trash/cheesie,
+/obj/item/trash/chips,
+/obj/item/trash/pistachios{
+	pixel_x = 6
+	},
+/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4{
+	dir = 9
+	},
+/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2{
+	dir = 9
+	},
+/obj/structure/cable/blue{
+	icon_state = "1-9"
+	},
+/turf/open/floor/plasteel,
+/area/ruin/beach/starwalker)
+"hV" = (
+/obj/structure/flora/ausbushes/fernybush,
+/turf/open/floor/plating/grass/beach/lit,
+/area/ruin/unpowered)
+"ia" = (
+/obj/machinery/atmospherics/components/unary/shuttle/heater{
+	dir = 1
+	},
+/obj/structure/window/reinforced,
+/obj/structure/window/reinforced{
+	dir = 1
+	},
+/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2{
+	dir = 4
+	},
+/obj/structure/spacevine/weak,
+/turf/open/floor/plating,
+/area/ruin/beach/starwalker)
+"id" = (
+/turf/open/floor/plating/beach/sand{
+	light_range = 2
+	},
+/area/ruin/unpowered)
+"it" = (
+/obj/effect/turf_decal/industrial/warning{
+	dir = 8
+	},
+/obj/structure/closet/body_bag,
+/obj/effect/decal/cleanable/dirt,
+/obj/effect/decal/cleanable/dirt,
+/obj/item/circuitboard/computer/rdconsole,
+/obj/item/circuitboard/machine/protolathe/department/science{
```

Comment:
it's just a circuitboard but is the science one appropriate for this ruin? I'd say like... Peacekeeping for ammo (pirate ship) or a cargo one.